app/views/admin/calendars/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>New Calendar</h1>
+<%= simple_form_for @calendar, url: admin_calendars_path(@calendar)  do |form| %>
+
+  <% if @calendar.calendar_type %>
+    <p>Regional Calendar</p>
+    <%= form.hidden_field :calendar_type, value: :regional %>
+  <% else %>
+    <%= form.input :calendar_type, as: :radio_buttons, collection: Calendar.calendar_types.keys, label_method: :humanize %>
+  <% end %>
+
+  <%= form.input :based_on_id, as: :select, collection: Calendar.national, include_blank: false %>
```

Comment:
Hmm, if we're forced to pick a national calendar then maybe this should just be a regional only form and we don't need the calendar_type select at all

Reply:
Yes, good point, it's inconsistent at the moment isn't it.